### DISCUSSION TITLE
Job Search | Interview Process

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#220] [Discuss a Topic]
This might apply to everyone at some point.

Hypothetical situation:  I've applied to my dream job, and I find a typo within their site. Is it bad form to let them know about it during the interview process or wait until I'm hired to bring it up? 

Would love your insight: @drteresavasquez @TrinityChristiana @ajawashington

**Comment by drteresavasquez** [comment]
@danicrosby I think this depends. I personally would not bring it up in the interview but would do so in the follow up email that I sent thanking them for their time and consideration. 

Why wouldn't I do it in the interview? You may be interviewed by the person who made the mistake and pointing out their flaws on an initial meeting may not be taken well especially if you don't know the person. AND you wouldn't want them to be that way with you on initial interview either. After you've gotten to know them in your questions about them and what they like about the company, etc it may open up the possibility to mention it in the follow up along with some things they shared. 

Mistakes are gonna happen so putting a bow on it can take away the identification of the error as well as show your attention to detail and your interest in their company. 

This is not legal advice and should not be misconstrued as such 😂

**Reply by danicrosby** [reply]
Perfectly said! And great points. Thank you!
